app/controllers/api/storage.php

```diff
@@ -24,9 +25,467 @@
+                    'size' => 2048,
+                    'signed' => true,
+                    'required' => false,
+                    'default' => null,
+                    'array' => false,
+                    'filters' => [],
+                ]),
+                new Document([
+                    '$id' => 'openSSLVersion',
+                    'type' => Database::VAR_STRING,
+                    'format' => '',
+                    'size' => 64,
+                    'signed' => true,
+                    'required' => false,
+                    'default' => null,
+                    'array' => false,
+                    'filters' => [],
+                ]),
+                new Document([
+                    '$id' => 'openSSLCipher',
+                    'type' => Database::VAR_STRING,
+                    'format' => '',
+                    'size' => 64,
+                    'signed' => true,
+                    'required' => false,
+                    'default' => null,
+                    'array' => false,
+                    'filters' => [],
+                ]),
+                new Document([
+                    '$id' => 'openSSLTag',
+                    'type' => Database::VAR_STRING,
+                    'format' => '',
+                    'size' => 2048,
+                    'signed' => true,
+                    'required' => false,
+                    'default' => null,
+                    'array' => false,
+                    'filters' => [],
+                ]),
+                new Document([
+                    '$id' => 'openSSLIV',
+                    'type' => Database::VAR_STRING,
+                    'format' => '',
+                    'size' => 2048,
+                    'signed' => true,
+                    'required' => false,
+                    'default' => null,
+                    'array' => false,
+                    'filters' => [],
+                ]),
+                new Document([
+                    '$id' => 'search',
+                    'type' => Database::VAR_STRING,
+                    'format' => '',
+                    'size' => 16384,
+                    'signed' => true,
+                    'required' => false,
+                    'default' => null,
+                    'array' => false,
+                    'filters' => [],
+                ]),
+            ], [
+                new Document([
+                    '$id' => '_key_bucket',
+                    'type' => Database::INDEX_KEY,
+                    'attributes' => ['bucketId'],
+                    'lengths' => [Database::LENGTH_KEY],
+                    'orders' => [Database::ORDER_ASC],
+                ]),
+                new Document([
+                    '$id' => '_key_search',
+                    'type' => Database::INDEX_FULLTEXT,
+                    'attributes' => ['search'],
+                    'lengths' => [2048],
+                    'orders' => [Database::ORDER_ASC],
+                ],),
```

Comment:
Isn't this a syntax error? It scares me. Can we get rid of it? 😅

Reply:
What are we talking about?